### ISSUE TITLE
Menu suite on Windows client

### ISSUE THREAD

**Issue by titofffffff** [Microsoft/AL#4074]
hello,

when I publish an extension the menu suite is modified and a link to the extension is added.
I could be good to have a property on extension to specify if we want add a link to the menu suite or not.

**Comment by AmolSalvi** [comment]
I don't think so MS will work on this because after 1 year or so Windows client will not be there and Web client doesn't have menu suite.
